.gitignore

```diff
@@ -16,6 +16,7 @@ pom.xml
 .nrepl-history
 .rebel_readline_history
 .idea
+*.iml
```

Comment:
If you want to adjust gitignore but not have it show up in commits, you can instead add things to `.git/info/exclude`, which is like gitignore but local to your copy of the project

Reply:
okay i will do this

Reply:
i'm fine with it being added here, there's no issue, i just wondered where it came from in the other pr.